packages/wdio-protocols/protocols/appium.json

```diff
@@ -1413,5 +1426,41 @@
         "description": "An object containing two fields: 'result', which is the return value of the script itself, and 'logs', which contains 3 inner fields, 'log', 'warn', and 'error', which hold an array of strings logged by console.log, console.warn, and console.error in the script's execution."
       }
     }
+  },
+  "/session/:sessionId/appium/events": {
+    "POST": {
+      "command": "getEvents",
+      "description": "Get events stored in appium server.",
+      "ref": "https://github.com/appium/appium-base-driver/pull/365",
+      "parameters": [{
+        "name": "type",
+        "type": "string[]",
+        "description": "Get events which are filtered with the type if the type is provided.",
+        "required": true
+      }],
+      "returns": {
+        "type": "object",
+        "name": "result",
+        "description": "A JSON hash of events like `{'commands' => [{'cmd' => 123455, ....}], 'startTime' => 1572954894127, }`."
+      }
+    }
+  },
+  "/session/:sessionId/appium/log_event": {
+    "POST": {
+      "command": "logEvent",
+      "description": "Store a custom event.",
+      "ref": "https://github.com/appium/appium-base-driver/pull/364",
```

Comment:
Would rather point to `https://github.com/appium/appium/blob/master/docs/en/commands/session/events/log-event.md`